op-node/rollup/driver/state.go

```diff
@@ -278,11 +280,15 @@ func (s *Driver) eventLoop() {
 				return
 			}
 			if s.network != nil && payload != nil {
-				// Publishing of unsafe data via p2p is optional.
-				// Errors are not severe enough to change/halt sequencing but should be logged and metered.
-				if err := s.network.PublishL2Payload(s.driverCtx, payload); err != nil {
-					s.log.Warn("failed to publish newly created block", "id", payload.ID(), "err", err)
-					s.metrics.RecordPublishingError()
+				if err := s.sequencerConductor.CommitUnsafePayload(s.driverCtx, payload); err != nil {
+					s.log.Error("failed to commit unsafe payload to conductor log", "id", payload.ID(), "err", err)
+				} else {
+					// Publishing of unsafe data via p2p is optional.
+					// Errors are not severe enough to change/halt sequencing but should be logged and metered.
+					if err := s.network.PublishL2Payload(s.driverCtx, payload); err != nil {
+						s.log.Warn("failed to publish newly created block", "id", payload.ID(), "err", err)
+						s.metrics.RecordPublishingError()
+					}
```

Comment:
nit, a style recommendation. This helps avoid nested if/else which makes code a little more readable

```suggestion
                if err := s.sequencerConductor.CommitUnsafePayload(s.driverCtx, payload); err != nil {
	                s.log.Error("failed to commit unsafe payload to conductor log", "id", payload.ID(), "err", err)
	                continue
                } 
                
                // Publishing of unsafe data via p2p is optional.
                // Errors are not severe enough to change/halt sequencing but should be logged and metered.
                if err := s.network.PublishL2Payload(s.driverCtx, payload); err != nil {
                    s.log.Warn("failed to publish newly created block", "id", payload.ID(), "err", err)
                    s.metrics.RecordPublishingError()
                }
```

Reply:
I'm pretty sure the continue applies to the `select` statement and will cause the [planSequencerAction() on line 294](https://github.com/ethereum-optimism/optimism/pull/8894/files/8aba66335eef22864b0c01d16ec10074e427f007#diff-511cbf4bb175ffd3df8b49e206073b0cc80140ecfe160a92bfd253cfb5728de2R294) to be skipped no?

Reply:
Right, this we can fix by putting `defer planSequencerAction` before the `if s.network != nil && payload != nil` check

Reply:
Just noting that this did not work, reverted.